cli.js

```diff
@@ -52,8 +52,12 @@ const throwSearchFail = (script, triedPaths) => {
 // Run
 // ----------------------------------------
 
-const scriptPath = argv._.shift()
-const scriptArgs = argv._.join(' ')
+const scriptPath = process.argv[2]
+const scriptArgs = process.argv.slice(3)
+  // for backward compatibility remove '--' first arg
+  // it was used as the first arg to separate script args from cli args
+  .filter((arg, i) => !(i === 0 && arg === '--'))
+  .join(' ')
```

Comment:
Here, we are using vanilla `process.argv` for getting our args.  We also trim the starting `--` if present, so old usage still works.

I left in use of `yargs` up top for the nice usage help and required args message.